lib/build-test-helper.js

```diff
@@ -16,7 +16,7 @@ module.exports = function (helperPath, cwd) {
 
 function buildUserHelper (helperPath, cwd) {
   if (helperPath && fs.existsSync(helperPath)) {
-    return _.assign({}, require(path.resolve(cwd, helperPath)), {
+    return _.assignIn({}, require(path.resolve(cwd, helperPath)), {
```

Comment:
Are we sure we want to assign inherited properties as well? (I don't know the answer. assuming assignIn is sufficient, but it _could_ be slower)